cmd/res/devices/camera.toml.example

```diff
@@ -0,0 +1,25 @@
+# Pre-defined Devices
+[[DeviceList]]
+Name = "Camera001"
+ProfileName = "onvif-camera"
+Description = "onvif conformant camera"
+  [DeviceList.Protocols]
+    [DeviceList.Protocols.Onvif]
+    Address = "192.168.12.123"
+    Port = "80"
+    # Assign AuthMode to "usernametoken" | "digest" | "both" | "none"
+    AuthMode = "digest"
+    SecretPath = "credentials001"
+
+# If having more than one camera, uncomment the following config settings
```

Comment:
Either comment out the block below or change the comment

Reply:
Comment out the block.